transcendence-app/src/user/user.domain.ts

```diff
@@ -1,9 +1,21 @@
 export class User {
+  username: string;
+  email: string;
+  avatar: string | null;
+  id: string;
+  createdAt: Date;
+
   constructor(
-    public username: string,
-    public email: string,
-    public avatar: string | null,
-    public id: string,
-    public created_at: number,
-  ) {}
```

Comment:
The [Swagger CLI](https://docs.nestjs.com/openapi/cli-plugin) plugin doesn't work with this class syntax. I've changed to a more verbose format to make it work and automatically detect the scheme of the class.

Alternatively, we could remove the constructor because I don't think we use it anywhere.

And we could define this class like this:
```typescript
export class User {	 
  username!: string;
  email!: string;
  avatar!: string | null;
  id!: string;
  createdAt!: Date;
}
```